src/minter-contract-utils/src/bft_bridge_api.rs

```diff
@@ -249,9 +249,57 @@ pub enum BridgeEvent {
 impl BridgeEvent {
     pub async fn collect_logs(
         evm_client: &EthJsonRpcClient<impl Client>,
+        mut from_block: u64,
+        blocks_to_fetch: u64,
+        bridge_contract: H160,
+    ) -> Result<Vec<Log>, anyhow::Error> {
+        const DEFAULT_BLOCKS_TO_COLLECT_PER_PAGE: u64 = 128;
+
+        let mut offset = DEFAULT_BLOCKS_TO_COLLECT_PER_PAGE;
+        let mut logs = Vec::new();
+
+        while from_block < blocks_to_fetch {
+            match Self::collect_logs_from_to(
```

Comment:
Is the block range inclusive? If yes, then the end block is fecthed twice as start block of the next loop